packages/manager/modules/logs-to-customer/src/components/services/KnowMoreLink.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Links, LinkType } from '@ovh-ux/manager-react-components';
+import { CountryCode } from '@ovh-ux/manager-config';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import {
+  KNOW_MORE_BASE_URL,
+  KNOW_MORE_LIST,
+} from '../subscriptions/SubscriptionEmpty.constants';
+
+const KnowMoreLink = () => {
+  const { t } = useTranslation('logSubscription');
+  const { shell } = useContext(ShellContext);
+  const { environment } = shell;
+  const [knowMoreLink, setKnowMoreLink] = useState('');
+
+  useEffect(() => {
+    const getSubSidiary = async () => {
+      const env = await environment.getEnvironment();
+      const { ovhSubsidiary } = env.getUser();
+      const guide =
+        KNOW_MORE_LIST[ovhSubsidiary as CountryCode] || KNOW_MORE_LIST.GB;
+      const knowMoreUrl =
+        KNOW_MORE_BASE_URL[ovhSubsidiary] ?? KNOW_MORE_BASE_URL.EU;
+      setKnowMoreLink(`${knowMoreUrl}${guide}`);
+    };
+    getSubSidiary();
+  }, []);
```

Comment:
Why don't use useQuery ?

```suggestion
  const { data: knowMoreLink = '', isLoading } = useQuery({
    queryKey: ['knowMoreLink'],
    queryFn: async () => {
      const env = await environment.getEnvironment();
      const { ovhSubsidiary } = env.getUser();
      const guide =
        KNOW_MORE_LIST[ovhSubsidiary as CountryCode] || KNOW_MORE_LIST.GB;
      const knowMoreUrl =
        KNOW_MORE_BASE_URL[ovhSubsidiary] ?? KNOW_MORE_BASE_URL.EU;
      return `${knowMoreUrl}${guide}`;
    },
  });
```

Reply:
I rather use useQuery for API calls only.